json_serializable/test/generic_files/generic_class.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2017, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// ignore_for_file: annotate_overrides
+
+import 'package:json_annotation/json_annotation.dart';
+
+part 'generic_class.g.dart';
+
+@JsonSerializable()
+class GenericClass<T extends num, S> extends Object
+    with _$GenericClassSerializerMixin<T, S> {
+  @JsonKey(fromJson: _dataFromJson, toJson: _dataToJson)
```

Comment:
are `_dataFromJson` and `_dataToJson` required for this to work?

Reply:
Yup. Will document as such.

Reply:
https://github.com/dart-lang/json_serializable/issues/171